File: 3110.score-of-a-string.java
/*
 * @lc app=leetcode id=3110 lang=java
 *
 * [3110] Score of a String
 */

// @lc code=start
class Solution {
    public int scoreOfString(String s) {
        int score = 0;
        for (int i = 0; i<s.length() - 1; i++) {
            int a = s.charAt(i) - s.charAt(i+1);
            score += Math.abs(a);
        }
        return score;
    }
}
// @lc code=end

File: 3110.score-of-a-string.py
#
# @lc app=leetcode id=3110 lang=python
#
# [3110] Score of a String
#

# @lc code=start
class Solution(object):
    def scoreOfString(self, s):
        """
        :type s: str
        :rtype: int
        """
        score = 0
        for i in range(len(s) - 1):
            score += abs(ord(s[i]) - ord(s[i+1]))
        return score
        
        
# @lc code=end

